### PR TITLE
Add multiple controller case

### PR DESCRIPTION
**Pull request by hsmj1412**
Only support PPC

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by hsmj1412** [comment]
Due to low ppc resources, I can not continue this week.

File: libvirt/tests/cfg/controller/multiple_controller.cfg [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    type = multiple_controller
+    start_vm = no
+    variants:
+        - positive_tests:
+            variants:
+                - controller_max:
+                    controller_type = pci
+                    controller_model = pci-root
+                    controller_num = 31
+                - without_index:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/controller/multiple_controller.cfg]: change to controller_without_index

File: libvirt/tests/cfg/controller/multiple_controller.cfg [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    controller_type = pci
+                    controller_model = pci-root
+                    controller_num = 3
+                    with_index = 'no'
+                - with_gap:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/controller/multiple_controller.cfg]: see previous comments,and following also has similar issues

File: libvirt/tests/cfg/controller/multiple_controller.cfg (hunk 1 above)
Comment (hsmj1412) [reply libvirt/tests/cfg/controller/multiple_controller.cfg]: done

File: libvirt/tests/cfg/controller/multiple_controller.cfg (hunk 2 above)
Comment (hsmj1412) [reply libvirt/tests/cfg/controller/multiple_controller.cfg]: done

**Comment by hsmj1412** [comment]
@chunfuwen Done

**Comment by prudmirya** [comment]
@hsmj1412 @chunfuwen y it is not merged? because of this my PR #1284 in hold.

**Comment by hsmj1412** [comment]
@prudmirya Why this pr blocked yours to be merged? the two prs are not in the same file

**Pull request merged**